src/components/RequestFilters.svelte

```diff
@@ -57,7 +58,12 @@ function onDestinationInput(event) {
 
 function onOriginInput(event) {
   const query = event.detail
-  setFilters({ origin: query })
+  setFilters({
+    fromDescription: (query !== null) && query.description,
```

Comment:
can `query` ever be `undefined` or empty...anything falsey that might have an undesired effect here?

Reply:
I dunno, but suspect not. Is the safe answer to put ` || ''` at the end?

Reply:
`query` can only be whatever value the `LocationFilter` dispatches, which is currently just whatever the `LocationInput` dispatches. That will be `null` when the LocationInput is cleared, or something along the lines of `{ description: ..., latitude: ..., longitude: ..., country: ... }` if a location is selected.